doc/source/management/data_at_rest_encryption.rst

```diff
@@ -80,6 +80,54 @@ Trying to add unencrypted table to this table space will result in an error:
   |Percona XtraBackup| currently doesn't support backup of encrypted general
   tablespaces.
 
+Binary log encryption
+=====================
+
+A new option, implemented since |Percona Server| :rn:`5.7.20-19`, is 
+encryption of binary and relay logs, triggered by the 
+:variable:`encrypt_binlog` variable.
+
+Besides turning :variable:`encrypt_binlog` ``ON``, this feature requires both
+`master_verify_checksum
+<https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_master_verify_checksum>`_ 
+and `binlog_checksum
+<https://dev.mysql.com/doc/refman/5.7/en/replication-options-binary-log.html#sysvar_binlog_checksum>`_ 
+variables to be turned ``ON``.
+
+Distinct keys are used for binary and relay logs, and key identifiers include
+server UUIDs, which makes multi-source replication possible.
+The encryption key is always automatically rotated at the start of each
+binary log file. Therefore configuring the automatic purging of old binary
+logs ensures that old key versions are not in use. 
+``PURGE BINARY LOGS`` and ``RESET MASTER`` commands delete obsolete binary log 
+key versions from the keyring, and obsolete relay log key versions are deleted 
+together with their corresponding relay logs. This scheme is independent from 
+the master instance key and its rotation. 
+
+While replication, master sends the stream of decrypted binary log events to a 
+slave (SSL connections can be set up to encrypt them in transport). That said, 
+masters and slaves are free to use differing keyring plugins.
+
+Decryption of binary logs can be done using ``mysqlbinlog`` with 
```

Comment:
Maybe "Dumping of encrypted binary logs", if yes, then the next note should be changed accordingly too